app/brave_settings_strings.grdp

```diff
@@ -14,23 +14,23 @@
 
   <!-- Settings / Get started -->
   <message name="IDS_SETTINGS_BRAVE_GET_STARTED_TITLE" desc="The title for the Get Started section in settings">
-    Get started
+    Get Started
```

Comment:
I believe our style guidelines have changed since the issue was created. The style guide indicates that we use sentence case. cc: @rmcfadden3

Reply:
Yeah, for desktop and Android this should be `Get started`